components/recurring-contributions/UpdatePaymentMethodPopUp.js

```diff
@@ -238,6 +239,7 @@ const UpdatePaymentMethodPopUp = ({ setMenuState, contribution, onCloseEdit, loa
   const [selectedPaymentMethod, setSelectedPaymentMethod] = useState(null);
   const [loadingSelectedPaymentMethod, setLoadingSelectedPaymentMethod] = useState(true);
   const [stripe, setStripe] = useState(null);
+  const [elements, setElements] = useState(null);
```

Comment:
Same here, we should make it clearer that this is about stripe

Reply:
Done. 😄